pod/perlfunc.pod

```diff
@@ -6156,6 +6156,24 @@ X<push> X<stack>
 
 =for Pod::Functions append one or more elements to an array
 
+Adds one or more items to the B<end> of an array.
+
+	my @animals = ("cat");
+	push(@animals, "mouse"); # ("cat", "mouse")
+
+	my @colors = ("red");
+	push(@colors, ("blue", "green"); # ("red", "blue", "green")
```

Comment:
Two open parens, one closing.

Reply:
There's a few of these, and in the unshift examples. Not sure I'd bother with the inner parens, the subs are variadic and the parens might confuse people? YMMV

Reply:
Good catch... I updated the PR